### PR TITLE
Update Get Started Guide structure

### PR DESCRIPTION
**Pull request by xu-yuan1**
### Details:
 - *update structure*
 - *add troubleshooting guide (WIP, to be updated by https://github.com/openvinotoolkit/openvino/pull/11785)*

### Tickets:
 - *https://jira.devtools.intel.com/browse/CVS-83936*

### CODE REVIEW

File: docs/install_guides/pypi-openvino-dev.md [hunk 1]
```diff
@@ -76,23 +55,38 @@ python -m pip install --upgrade pip
...
+```sh
+pip install openvino-dev[extras]
+```
+ where `extras` has the following values: 
+
+| Extras Value                    | DL Framework                                                                     |
+| :-------------------------------| :------------------------------------------------------------------------------- |
+| caffe                           |   [Caffe*](https://caffe.berkeleyvision.org/)                                    |
+| kaldi                           |   [Kaldi*](https://github.com/kaldi-asr/kaldi)                                   |
+| mxnet                           |   [MXNet*](https://mxnet.apache.org/)                                            |
```
Comment (kblaszczak-intel) [inline docs/install_guides/pypi-openvino-dev.md:79]:
```suggestion
| mxnet                           |   [Apache MXNet*](https://mxnet.apache.org/)                                            |
```

File: docs/install_guides/pypi-openvino-dev.md [hunk 2]
```diff
@@ -76,23 +55,38 @@ python -m pip install --upgrade pip
...
+| pytorch                         |   [PyTorch*](https://pytorch.org/)                                               |
+| tensorflow                      |   [TensorFlow* 1.x](https://www.tensorflow.org/versions#tensorflow_1)            |
+| tensorflow2                     |   [TensorFlow* 2.x](https://www.tensorflow.org/versions#tensorflow_2)            |
 
 For example, to install and configure the components for working with TensorFlow 2.x, MXNet and Caffe, use the following command:  
    ```sh
    pip install openvino-dev[tensorflow2,mxnet,caffe]
    ```
-**NOTE**: Support of MO in TensorFlow 1.x environment is deprecated. Use TensorFlow 2.x environment to convert both TensorFlow 1.x and 2.x models
+> **NOTE**: Support of Model Optimizer in TensorFlow 1.x environment is deprecated. Use TensorFlow 2.x environment to convert both TensorFlow 1.x and 2.x models.
```
Comment (kblaszczak-intel) [inline docs/install_guides/pypi-openvino-dev.md:89]:
```suggestion
> **NOTE**: Model Optimizer support for TensorFlow 1.x environment has been deprecated. Use TensorFlow 2.x environment to convert both TensorFlow 1.x and 2.x models.
```

**Review by kblaszczak-intel — APPROVED** [review]
Added some comments - address them at your discretion.

File: docs/install_guides/installing-model-dev-tools.md [hunk 3]
```diff
@@ -18,9 +18,9 @@ While installing OpenVINO Development Tools, OpenVINO Runtime will also be insta
 
 ### Step 1. Set Up Python Virtual Environment
 
-To avoid dependency conflicts, use a virtual environment. Skip this step only if you do want to install all dependencies globally.
+Use a virtual environment to avoid dependency conflicts.
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/installing-model-dev-tools.md:21]: yes, that's mentioned in Pawel's ticket that we should prevent users to install dependencies globally.

File: docs/install_guides/installing-openvino-overview.md [hunk 4]
```diff
@@ -1,4 +1,17 @@
-# Overview {#openvino_docs_install_guides_overview}
+# Installing Intel® Distribution of OpenVINO™ Toolkit {#openvino_docs_install_guides_overview}
+
+@sphinxdirective
+
+.. toctree::
+   :maxdepth: 3
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-overview.md:6]: aha, I thought its child pages should have at least 3 levels, not sure if it only reflects the current page depth. but anyway it seems this value is neglected by Sphinx.

[the remark this answers is not in the document]
Comment (kblaszczak-intel) [reply docs/install_guides/installing-openvino-overview.md:6]: Oh, I see. I guess for the menu it does not matter and is relevant only if we want to include the table of contents in the page itself. We should probably discuss its usage for the new docs repository.

**Pull request merged**